qualtran/surface_code/physical_parameters.py

```diff
@@ -0,0 +1,50 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from attrs import field, frozen
+
+
+@frozen
+class PhysicalParameters:
+    """The physical properties of a quantum computer..
+
+    Attributes:
+        t_gate_ns: Clifford gate time.
+        t_meas_ns: Measurement time.
```

Comment:
are these the times required to do a physical T gate and physical measurement? Or a logical T gate and measurement? Please clarify with a docstring

Reply:
`t` stands for time not T

Reply:
ok. Are they the physical times or logical times?

Reply:
physical

Reply:
can you please note this in the docstring

Reply:
done